.github/files/build-reminder-comment/check-test-reminder-comment.js

```diff
@@ -35,22 +46,25 @@ async function isTouchingSomethingNeedingTesting( github, owner, repo, number, c
  * @param {github} github  - Pre-authenticated octokit/rest.js client with pagination plugins
  * @param {object} context - Context of the workflow run
  * @param {core}   core    - A reference to the @actions/core package
- * @returns {Promise<number>} Promise resolving to a comment ID, or 0 if no comment is found.
+ * @returns {Promise} Promise resolving to an object with the following properties:
+ * - {commentId} - a comment ID, or 0 if no comment is found.
+ * - {projects} - an array of project strings needing testing.
  */
 async function checkTestReminderComment( github, context, core ) {
 	const { repo, issue } = context;
 	const { owner, repo: repoName } = repo;
 	const { TEST_COMMENT_INDICATOR } = process.env;
+	const data = {};
 
-	// Check if one of the files modified in this PR has been de-fusioned,
-	// and thus must now be tested on WordPress.com.
+	// Check if one of the files modified in this PR need testing on WordPress.com.
 	const touchesSomethingNeedingTesting = await isTouchingSomethingNeedingTesting(
 		github,
 		owner,
 		repoName,
 		issue.number,
 		core
 	);
+	data.projects = touchesSomethingNeedingTesting;
 
 	core.info(
 		`Build: This PR ${
```

Comment:
The use of `touchesSomethingNeedingTesting` in this message needs updating. In JavaScript, the empty array is truthy, not falsey like in PHP.

Reply:
Does c8432b1 look better?

Reply:
Yes, thanks.